compiler/input.c

```diff
@@ -0,0 +1,4 @@
+int main() {
```

Comment:
I don't see why you are adding this, maybe our input files should not be tracked by git?